src/lucky/params.cr

```diff
@@ -212,6 +219,14 @@ class Lucky::Params
     multipart_files[key.to_s]?
   end
 
+  def get_all_files(key : String | Symbol) : Array(Lucky::UploadedFile)
+    get_all_files?(key) || raise Lucky::MissingParamError.new(key.to_s)
+  end
+
+  def get_all_files?(key : String | Symbol) : Array(Lucky::UploadedFile)
+    multipart_files.fetch_all(key.to_s)
```

Comment:
This doesn't actually return nil which means the previous method would never actually raise anything... I only kept it for consistency between the methods. Maybe this could return nil if the array is empty?